pkg/pushsync/pushsync_test.go

```diff
@@ -145,20 +145,27 @@ func TestHandler(t *testing.T) {
 	triggerPeer := swarm.MustParseHexAddress("6000000000000000000000000000000000000000000000000000000000000000")
 	closestPeer := swarm.MustParseHexAddress("f000000000000000000000000000000000000000000000000000000000000000")
 
+	// mock call back function to see if pss message is delivered when it is received in the destination (closestPeer in this testcase)
+	hookWasCalled := make(chan bool, 1) // channel to check if hook is called
+	pssDeliver := func(ch swarm.Chunk) error{
```

Comment:
this line also needs gofmt (space before `{`)

Reply:
here too i see it in my code...

Reply:
same answer as before.